package.json

```diff
@@ -110,6 +110,10 @@
             {
                 "command": "extension.validateTdeTemplate",
                 "title": "MarkLogic: Validate TDE template"
+            },
+            {
+                "command": "extension.tdeExtractNodes",
+                "title": "MarkLogic: Extract data from TDE"
```

Comment:
How about "Extract data via TDE" - "from TDE" to me sounds like I'm extracting data from the TDE itself, but I'm really using the TDE to extract data from some other thing.